### PR TITLE
fix: added docstrings for status

### PR DESCRIPTION
**Pull request by abhilash-kumar-nair**
This PR adds docstrings for Status

### CODE REVIEW

**Review by iakovn — CHANGES_REQUESTED** [review]
Please add documentation on the `run_info.status` property explaining that users should instead get the operation status (and not case/experiment entity status) for tracing status changes of a running simulation.

**Comment by abhilash-kumar-nair** [comment]
> Please add documentation on the `run_info.status` property explaining that users should instead get the operation status (and not case/experiment entity status) for tracing status changes of a running simulation.

@iakovn Ah I had missed that. Fixed it. Thanks!

File: modelon/impact/client/entities/case.py [hunk 1]
```diff
@@ -53,7 +53,23 @@ def __init__(
 
     @property
     def status(self) -> CaseStatus:
-        """Status info for a Case, its type is CaseStatus."""
+        """
+        Status info for a Case.
+        Note: For tracing status changes of a running case execution,
```
[the remark this answers is not in the document]
Comment (abhilash-kumar-nair) [reply modelon/impact/client/entities/case.py]: I mainly looked at help() function docs. But I agree it makes sense to make it look nice in RTD too. Fixed it :)

**Review by iakovn — APPROVED** [review]
Nice improvement! 👍

**Pull request merged**